src/components/trends/smbg/FocusedSMBGRangeLabels.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes } from 'react';
+import Tooltip from '../common/Tooltip';
+
+import { MGDL_UNITS, MMOLL_UNITS } from '../../../utils/constants';
+import { displayBgValue } from '../../../utils/format';
+import { THREE_HRS, millisecondsAsTimeOfDay } from '../../../utils/datetime';
+
+import styles from './FocusedSMBGRangeLabels.css';
+
+const ONE_AND_A_HALF_HOURS = THREE_HRS / 2;
+
+const FocusedSMBGRangeLabels = (props) => {
+  const { focusedRange } = props;
+  if (!focusedRange) {
+    return null;
+  }
+
+  const { bgUnits, focusedRange: { data, position } } = props;
+  const timeFrom = millisecondsAsTimeOfDay(data.msX - ONE_AND_A_HALF_HOURS);
+  const timeTo = millisecondsAsTimeOfDay(data.msX + ONE_AND_A_HALF_HOURS);
```

Comment:
This breaks the flexibility to set the bin size to something other than three hours. Please follow the model [here](https://github.com/tidepool-org/viz/blob/master/src/utils/trends/data.js#L59) from the cbg code to add `msFrom` and `msTo` attributes based on the `binSize` in the `calculateSmbgStatsForBin` method.